docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -152,55 +152,49 @@ Closes all alerts that match the exception's conditions and were generated only
 
 [float]
 [[endpoint-rule-exceptions]]
-=== Add Elastic Endpoint Security exceptions
+=== Add Elastic {endpoint-sec} exceptions
 
-Like detection rule exceptions, you can add Endpoint agent exceptions via both
-the Elastic Endpoint Security rule and its generated alerts. Alerts generated
-from the Elastic Endpoint Security rule have the following fields:
+Like detection rule exceptions, you can add Endpoint agent exceptions either by editing the Elastic {endpoint-sec} rule or as an action on an alert generated by the {endpoint-sec} rule. Elastic {endpoint-sec} alerts have the following fields:
```

Comment:
```suggestion
Like detection rule exceptions, you can add Endpoint agent exceptions either by editing Elastic {endpoint-sec} rules or by adding them as actions on alerts generated by {endpoint-sec} rules. Elastic {endpoint-sec} alerts have the following fields:
```
Not sure about this one from a product standpoint or otherwise, but my intuition was that the plural "exceptions" meant that "rule" and "action" should also be plural.

Reply:
I think that works for everything except "actions" -- making that plural sounds like there's more than one action available in the UI for this.